rosa/cli.py

```diff
@@ -51,14 +54,21 @@ def is_logged_in(aws_region=None, allowed_commands=None):
 
 
 def execute_command(command):
+    def _wait_for_command_execution(_command):
+        for result in TimeoutSampler(
+            wait_timeout=TIMEOUT_5MIN,
+            sleep=SLEEP_1SEC,
+            func=lambda: subprocess.run(_command, capture_output=True, text=True),
```

Comment:
```suggestion
            func=subprocess.run,
            args=_command,
            capture_output=True,
            text=True,
```

No need to use lambda, you can pass the function with arguments separated, see example:
https://github.com/RedHatQE/managed-services-integration-tests/blob/9c41348e1587f4dd72a5a751c790595391db8cfe/tests/managed_services/mas_debezium/conftest.py#L107

Reply:
Tried at first but failed with:
```
    if _func.__name__ == "<lambda>":
AttributeError: 'CompletedProcess' object has no attribute '__name__'
```
Maybe it's diffetent because in subprocess.run, the argument _command is actually *popenargs [so it's expecting lambda](https://github.com/RedHatQE/openshift-python-wrapper/blob/main/ocp_resources/utils.py#L95)
Any ideas?